pkgs/dart_tooling_mcp_server/test/dart_tooling_mcp_server_test.dart

```diff
@@ -151,8 +154,34 @@ void main() {
   group('dart cli', () {
     late Tool dartFixTool;
     late Tool dartFormatTool;
+    late String copyDartCliAppsBin;
+    const copyDirectoryName = 'bin_copy';
+
+    Future<void> copyDartCliAppAndVerifyContents() async {
+      // Copy the original `dart_cli_app` directory so that changes from these
```

Comment:
Consider just copying the entire test fixture dir, but to a temp directory.

That way if a test gets into an unrecoverable state that has to be force killed you don't have to manually clean up this dir.

Reply:
For whatever reason I was running into issues with temporary directories. I originally tried to copy the entire `dart_cli_app` dir to a temporary directory but then the `dart fix` command wouldn't detect any fixes. Idk if this is because there wasn't a pubspec.yaml anywhere or what, but I eventually just went back to copy in place. I will add to .gitignore and add a todo in the test in case this becomes an issue later